napari/_qt/qt_event_loop.py

```diff
@@ -133,11 +136,12 @@ def get_app(
         app.setWindowIcon(QIcon(kwargs.get('icon')))
         set_app_id(kwargs.get('app_id'))
 
-        if os.getenv("NAPARI_CATCH_ERRORS") not in ('0', 'False'):
-
-            notification_manager.notification_ready.connect(
-                NapariQtNotification.show_notification
-            )
+        notification_manager.notification_ready.connect(
```

Comment:
we always hook it up now, whether to show or not is determined inside of `show_notification` based on the current settings level